aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -881,11 +898,28 @@ Tensor minimum(const Tensor& self, const Tensor& other) {
 
 // binary min, alias for minimum
 Tensor& min_out(Tensor& result, const Tensor& self, const Tensor& other) {
-  return at::minimum_out(result, self, other);
+    return at::minimum_out(result, self, other);
 }
 
 Tensor min(const Tensor& self, const Tensor& other) {
-  return at::minimum(self, other);
+    return at::minimum(self, other);
+}
+
+Tensor& fmin_out(Tensor& result, const Tensor& self, const Tensor& other) {
```

Comment:
The result tensor must be the last parameter (see comments on native_functions.yaml)

Reply:
Addressed!